docs/docs/deploy/index.md

```diff
@@ -9,10 +9,84 @@ description: >
 ---
 ---
 
-Firezone can be deployed on most popular Linux distributions in a few minutes.
+Firezone can be self-hosted on a server running a supported
+[Linux distribution]({% link docs/deploy/supported-platforms.md %})
+in a few minutes. This guide will walk you through the steps to get started.
+
+## Step 1: Environment Setup
+
+### Supported operating systems
+
 Start by checking if your environment is listed on
 [supported platforms]({% link docs/deploy/supported-platforms.md %}).
 A kernel upgrade may be required to ensure WireGuard® is available.
 
-\
-[Start Here: Supported Platforms]({%link docs/deploy/supported-platforms.md%}){:.btn.btn-purple}
+### Firewall settings
+
+Firezone requires ports `80/tcp`, `443/tcp`, and `51820/udp` to be open.
```

Comment:
This sort of contradicts what we say on the security considerations page: https://docs.firezone.dev/docs/administer/security-considerations/

Also, I'd reword this to be clear that typically `80/tcp` or `443/tcp` need to be available, not both. We highly recommend `443/tcp` and by default will redirect http requests to be https.

Also these can change so should clarify "by default"

Also they'll need to make sure forwarding is enabled in their firewall -- see the debugging https://docs.firezone.dev/docs/administer/troubleshoot/#debugging-connectivity-issues